st2chatops/files/st2.chatops.env

```diff
@@ -0,0 +1,133 @@
+export st2_hostname="${st2_hostname:-localhost}"
```

Comment:
Don't need to copy-paste the entire `st2chatops.env`.
Instead can source the one which comes with the `st2chatops` package under `/opt/stackstorm/chatops/st2chatops.env`.

Then by passing/providing ENV variables in Dockerfile it will override defaults coming from the st2chatops.env like this https://github.com/StackStorm/st2chatops/blob/c99d779d6e67385156bfd11b4c9997cf7e6f7ca8/st2chatops.env#L25-L27